.ruby-version

```diff
@@ -1 +0,0 @@
-2.7.0
```

Comment:
Mind leaving this file in? Helps keeping local development environments closer in version parity.

Reply:
Saw your note about Federalist, let's open a bug with them.

Reply:
Let's. From my understanding, when we specify a 2.7.0 ruby version, a new ruby is built from scratch... every time. (Yes, 2.7.0 should be already available as a pre-built binary, I have no idea why it's not). 

For now, let's add this back -- but with say, 2.6.6 instead? (Or whatever version Federalist is using by default)

Reply:
Federalist default is good, thanks.